lib/navigation/preprocess_encoders.cpp

```diff
@@ -1,56 +1,103 @@
+#include "consts.hpp"
 #include "preprocess_encoders.hpp"
 
+#include "core/types.hpp"
+
 namespace hyped::navigation {
 
-EncodersPreprocessor::EncodersPreprocessor()
+EncodersPreprocessor::EncodersPreprocessor(core::ILogger &logger)
+    : logger_(logger),
+      num_consecutive_outliers_per_encoder_({0, 0, 0, 0}),
+      is_reliable_per_encoder_({true, true, true, true}),
```

Comment:
Actually, the name the other PR went with is different so let's change this for consistency.

`are_encoders_reliable_`